base/dcp_feed.go

```diff
@@ -129,7 +135,7 @@ func (r *DCPReceiver) GetBucketNotifyFn() sgbucket.BucketNotifyFn {
 }
 
 func (r *DCPReceiver) OnError(err error) {
-	Warnf(KeyAll, "Error processing DCP stream - will attempt to restart/reconnect if appropriate: %v.", err)
+	r.Warnf(KeyAll, "Error processing DCP stream %s - will attempt to restart/reconnect if appropriate: %v.", r.feedID, err)
```

Comment:
```suggestion
	r.Warnf(KeyAll, "Error processing DCP stream - will attempt to restart/reconnect if appropriate: %v.", err)
```

Already covered by receiver-specific logging

Reply:
Thanks - will fix.